end-to-end-test/local/docker/Dockerfile.screenshottest

```diff
@@ -0,0 +1,10 @@
+FROM circleci/node:8.12.0-browsers
```

Comment:
Why do we need this image exactly? To have a `circleci/node:8.12.0-browsers` + `yarn 1.13.0` image?

Reply:
Yes, this is the environment in which the actual screenshots are run.